airflow/auth/managers/base_auth_manager.py

```diff
@@ -61,7 +61,7 @@
 
 # This cannot be in the TYPE_CHECKING block since some providers import it globally.
 # TODO: Move this inside once all providers drop Airflow 2.x support.
-ResourceMethod = Literal["GET", "POST", "PUT", "DELETE", "MENU"]
+ResourceMethod = Literal["GET", "POST", "PUT", "PATCH", "DELETE", "MENU"]
```

Comment:
Why is `PATCH` required? Is it to support future changes?

Reply:
yeah - https://github.com/apache/airflow/pull/47062#discussion_r1971338292